src/controllers/Game.ts

```diff
@@ -154,20 +131,33 @@ export default class Game {
     }
 
 
-    async getScoresaber(playerId: string): Promise<any> {
+    async getScoresaber(playerId: string): Promise<ScoreSaberPlayerInfo | null> {
         try {
-            const response = await fetch(`https://scoresaber.com/api/player/${playerId}/full`);
-            if (response.status === 404) {
-                return {};
+            const response = await got.get(`https://scoresaber.com/api/player/${playerId}/full`);
+            switch (response.statusCode) {
```

Comment:
This is pretty much an if-else at this point

Reply:
The switch was put there in case of other status checks in the future, up to you if revert

Reply:
I don't see us using any other status checks, either you got the data or you did not xD.